refactron/cli.py

```
@@ -419,5 +419,177 @@ def init() -> None:
+
+        # Get suggestions
+        suggestions = refactron.get_ai_suggestions(target)
+
+        console.print("[bold green]📖 Code Explanation:[/bold green]")
+        console.print(suggestions["explanation"])
+        console.print("\n[bold yellow]💡 Improvement Suggestions:[/bold yellow]")
+        console.print(suggestions["improvements"])
+        console.print("\n[bold cyan]⚡ Optimization Recommendations:[/bold cyan]")
+        console.print(suggestions["optimizations"])
+
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)